### ISSUE TITLE
[Runtime] Support player pose synchronization (or rather, subscription to specific transforms from an actor)

### ISSUE THREAD

**Issue by eanders-ms** [Microsoft/mixed-reality-extension-sdk#431]
Not all platforms have the capability to synchronize player skeletons or other data. Add an option to the runtime for synchronizing data between clients. A generic p2p rpc, essentially. This would enable clients to sync skeleton updates.

**Comment by sorenhan** [comment]
I don't think this should be implemented as suggested. IMO host apps are responsible for general pose synchronization, and only in one specific case does it make sense (see #495)
